### ISSUE TITLE
GDH not working in Party version - unknown why

### ISSUE THREAD

**Issue by patient87** [GeBo1/GeBoPlugins#5]
Hi, sorry, this is the most appropriate way I could figure out how to let you know that as of HF Patch 3.8, Game Dialog Helper isn't working. I even tried manually installing and updating the API, Bepis, etc, and still it doesn't work. Is there maybe a conflict or something I'm missing? Any help would be appreciated, Discord is devoid. Thanks.

**Comment by GeBo1** [comment]
Can you explain what "doesn't work" means?  What settings are you using, what's happening, and what do you expect to have happen?

**Comment by patient87** [comment]
I'm sorry. I've installed HF Patch 3.8, and none of the functions of the dialog helper are working as intended. No matter which settings I choose, none of the dialog options are amended, nor colors changed, regardless of relationship setting. I've even tried reinstalling HF Patch, and then manually installing all newest releases of prerequisites and of the helper itself. No luck.

**Comment by GeBo1** [comment]
Changing the settings (especially Plugin Mode) on the fly may not take immediate effect. You may need to restart the game. Are you restarting the game when changing settings?  Also for the Advanced Game Logic mode (default) it only shows you anything if you've previously guessed that answer, passed the check to remember it, and passed the check to recall your previous answer, so I'd expect you not see much of anything in this mode for a bit.

That said, try the following to see if it's working at all.

- Ensure you have [GameDialogHelper v1.0.1](https://github.com/GeBo1/GeBoPlugins/releases/tag/r23)
- Start the game and go into settings, and set the following:
    - Plugin Mode: Relationship
    - Minimum Relationship: Anyone
- Quit the game
- Restart and then start/load a game
- Talk to someone and choose one of the dialogs that has choices

**Comment by patient87** [comment]
I followed the steps you listed, and it still does not appear to be functioning. Do the choices still highlight when you're at max relationship? Do I need to start a new game? A previous version (from hf patch 3.5 I believe) worked fine before updating to 3.8.

**Comment by Afrokun** [comment]
Following this, as I'm having the same issues.

**Comment by GeBo1** [comment]
This appears to be a Party only issue. Not sure what's causing it yet.

**Comment by Mortmin** [comment]
Still an issue with the Party version.
Providing my log, mentioning that it is loading, but giving some message about XML: 35, when I come to the talk scene
[output_log.txt](https://github.com/GeBo1/GeBoPlugins/files/7602945/output_log.txt)
It was set to Relationship Mode, for anyone, and set to change color.

**Comment by GeBo1** [comment]
> Providing my log, mentioning that it is loading, but giving some message about XML: 35, when I come to the talk scene

That's just saying it loaded 35 questions without issue, so there's no errors in the log, unfortunately.

**Comment by Xplorer96** [comment]
Hi, I'm unsure if this mod is still being maintained or not, if not, sorry for the thread necro.

I've just started playing Koikatsu Party yesterday and after 8hours of playtime, the correct dialogue options still aren't shown, regardless if I chose append text or color change (might be my colorblind problem though if the red and green isn't very distinct).

I've tried changing the keybinds, no result. Changing the options to "Relationship Based" and "Min. relationship: Anyone", still no result.

I have tried asking in the Koikatsu category of the Illusion Soft Discord and have gotten no reply still (it seems dead). Thanks.

**Comment by patient87** [comment]
One of the guys who worked on the mods told me it has to do with the us
version vs Japanese version. So if you have one, try installing the other
and installing the mod packs to it. I can't remember which version I had,
and which version the mods were designed to work properly for. The versions
have different root folder names or something. Godspeed.

**Comment by miquelfire** [comment]
It's the US version that is currently broken. This mod works for the Japanese version, but used to work for the US version. I'm not sure, but an older version of the mod might still work with both.

**Comment by miquelfire** [comment]
Note, the reason an older version works might be related to a change in the US version that broke all versions, but the version of this mod that broke was released at the same time. If when the mod broke was purely the mod's fault, then any version before 1.0 should work. If the US version was updated in a way that broke this mod, then it won't matter which version you load.

**Comment by Xplorer96** [comment]
> Note, the reason an older version works might be related to a change in the US version that broke all versions, but the version of this mod that broke was released at the same time. If when the mod broke was purely the mod's fault, then any version before 1.0 should work. If the US version was updated in a way that broke this mod, then it won't matter which version you load.

I see, thank you for the reply, I've asked in 2 Koikatsu Discords and had no reply from them so I was about to give up on it and take it as not maintained anymore. I shall try what you suggested, tyvm~

**Comment by GeBo1** [comment]
Yes, the issue is still open because it’s not fixed, and I’m aware that it’s not. I check the discords, but I’m not on there every day. The party version handles the options quite differently and I haven’t had the time to work it out yet. 

No one pinged me on the discords, so I'm never going to notice a random message.

**Comment by GeBo1** [comment]
Should be able to use the ideas from #11 to make a version that works in both party and original game.
